src/Controllers/Home/Controller.php

```diff
@@ -17,41 +19,102 @@ class Controller
         LoggerAwareTrait;
 
     private
+        $applications,
         $twig,
         $servers;
 
-    public function __construct(\Twig_Environment $twig, array $servers)
+    public function __construct(\Twig_Environment $twig, array $servers, Configuration $config)
     {
         $this->twig = $twig;
         $this->servers = $servers;
 
         $this->logger = new NullLogger();
+
+        $this->applications = $this->populateApplications($config);
+    }
+
+    private function populateApplications(Configuration $config)
+    {
+        if($config->read('process/applications/enabled', false))
+        {
+            $apps = [];
+
+            foreach($this->servers as $server)
+            {
+                $apps = array_merge($apps, $server->extractApplicationList());
+            }
+
+            $apps = array_unique($apps);
+
+            return $apps;
+        }
+
+        return false;
     }
 
     public function homeAction()
     {
+        $processes = [];
+        foreach($this->servers as $server)
+        {
+            $processes = array_merge($processes, $server->getProcessList());
+        }
+
+        return $this->twig->render('home.twig', [
+            'servers' => $this->servers,
+            'apps' => $this->applications,
+            'processes' => $processes,
+        ]);
+    }
+
+    public function serversAction($serverName)
+    {
+        $server = $this->servers[$serverName];
+
+        return $this->twig->render('home.twig', [
+            'servers' => $this->servers,
+            'apps' => $this->applications,
+            'processes' => $server->getProcessList(),
+        ]);
+    }
+
+    public function applicationsAction($applicationName)
+    {
+        $processes = [];
+        foreach($this->servers as $server)
+        {
+            $processes = array_merge($processes, $server->getProcessList());
+        }
+
+        $processes = new ApplicationFilterIterator(
```

Comment:
TODO :  enabled/disabled in the configuration (and the view by server also maybe)